src/components/Patient/PatientRegistration.tsx

```
@@ -471,7 +471,7 @@ export default function PatientRegistration(
                   ["dob", t("date_of_birth")],
                   ["age", t("age")],
                 ].map(([key, label]) => (
-                  <TabsTrigger value={key}>{label}</TabsTrigger>
+                  <TabsTrigger value={key as string}>{label}</TabsTrigger>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)